lib/gis/file_name.c

```diff
@@ -27,8 +27,29 @@ static void append_char(char*, char);
   If <i>name</i> is of the form "nnn@ppp" then path is set as if name
   had been "nnn" and mapset had been "ppp" (mapset parameter itself is
   ignored in this case).
-  
-  \param[out] path buffer to hold resultant full path to file
+
+  Paths to files are in form:
+  /home/user/GISDBASE/LOCATION/mapset/element/name
+
+  path input buffer memory must be preallocated by caller.
+
+  C:
+  @code
+  char path[GPATH_MAX];
+  @endcode
+  Python:
+  @code
+  import ctypes
+  from grass.pygrass.utils import decode
+  from grass.lib.gis import G_file_name, GPATH_MAX
+
+  path = ctypes.create_string_buffer(GPATH_MAX)
+  path_str = decode(G_file_name(path, "elem", "name", "mapset"))
+  print(path_str)
+  /home/user/gisdb/loc/mapset/elem/name
```

Comment:
Python examples with outputs are typically written with the `>>>` prompt. Alternatively, a cross-language style would be `# prints /home/user/` or `# path_str now contains ...`

Reply:
Thanks for a hint!